examples/tensorflow/license-plate-reader/README.md

```diff
@@ -24,16 +24,53 @@ Out of these three models (*YOLOv3*, *CRAFT* and *CRNN*) only *YOLOv3* has been
 
 The other two models, *CRAFT* and *CRNN*, can be found in [keras-ocr](https://github.com/faustomorales/keras-ocr).
 
-## Deploying
+## Deployment - Lite Version
 
-The recommended number of instances to run this smoothly on a video stream is about 12 GPU instances (2 GPU instances for *YOLOv3* and 10 for *CRNN* + *CRAFT*). `cortex.yaml` is already set up to use these 12 instances. Note: this is the optimal number of instances when using the `g4dn.xlarge` instance type. For the client to work smoothly, the number of workers per replica can be adjusted, especially for `p3` or `g4` instances, where the GPU has a lot of compute capacity.
+A lite version of the deployment is available with `cortex_lite.yaml`. A single GPU is required for this deployment (i.e. `g4dn.xlarge`).
```

Comment:
I think we should add a brief description to lite version explaining that it reads license plates on a single image

Reply:
Done!